### ISSUE TITLE
Request for Trained model on Ravdess dataset

### ISSUE THREAD

**Issue by choise28** [katerynaCh/multimodal-emotion-recognition#2]
We are referencing your paper "Self-attention fusion for audiovisual emotion recognition with incomplete data, ICPR 2022" in our research.

And, we want to apply facial expression recognition with code shared at "https://github.com/katerynaCh/multimodal-emotion-recognition".

Can you share the trained model on Ravdess dataset?

It will be of great help to our research.

I look forward to hearing from you.

**Comment by lindalibjchn** [comment]
> We are referencing your paper "Self-attention fusion for audiovisual emotion recognition with incomplete data, ICPR 2022" in our research.
> 
> And, we want to apply facial expression recognition with code shared at "https://github.com/katerynaCh/multimodal-emotion-recognition".
> 
> Can you share the trained model on Ravdess dataset?
> 
> It will be of great help to our research.
> 
> I look forward to hearing from you.

I have the same request.

**Comment by katerynaCh** [comment]
Hi @choise28 @lindalibjchn , than you for your interest. I will provide some pretrained models shortly

**Comment by lindalibjchn** [comment]
> Hi @choise28 @lindalibjchn , than you for your interest. I will provide some pretrained model

Thanks in advanced, I am looking forward to in.

**Comment by katerynaCh** [comment]
I have uploaded three models with each attention type and modality dropout: https://tuni-my.sharepoint.com/:f:/g/personal/kateryna_chumachenko_tuni_fi/EvPvmdroOg1Hgtsvxo6N9yMBgC9nHjo-V1FVHwzcf8FTqw?e=GemHm1. The split for these models is: test: 5,10,16,19; val: 11,13,18,22; train: the rest. Please note the RAVDESS license before using these models.

If you are interested in models with other configurations, let me know and I can upload them as well
